linux/installer/common/psw/Makefile

```diff
@@ -63,6 +63,9 @@ install:
 	mv $(PSW_LIB_PATH)/libsgx_enclave_common.so $(USR_LIB_PATH)
 	rmdir $(PSW_LIB_PATH)
 	cd $(USR_LIB_PATH) && \
+	mv libsgx_urts.so libsgx_urts.so.$(URTS_FULL_VER) && \
+	ln -fs libsgx_urts.so.$(URTS_FULL_VER) libsgx_urts.so.$(URTS_MAJOR_VER) && \
```

Comment:
Bbetter to remove `URTS_MAJOR_VER` and call `SPLIT_VERSION` as the other versions.